samples/app-dev/Connectors/MessagingAndEventPubSub/PulsarSample/PulsarChunkingAndBatching/readme.md

```diff
@@ -0,0 +1,127 @@
+# Flogo Application sample for Apache Pulsar Connector
+
+
+## Description
+
+This samples demonstrate a simple worflow about how pulsar publisher and consumer work through a flogo application using the Chunking and batching feature.
+
+The sample application has two flows: the first is 'Pulsar-producer', which publishes messages to a persistent Pulsar topic using the chunking feature and returns the payload of published messages, and the other flow has a Pulsar consumer trigger, which listens to the same topic using a Pulsar subscription and aggregates all the chunks and receives the original message.And for batching Pulsar-producer', which publishes messages in batches to a persistent Pulsar topic and returns the payload of published messages, and the other consumer app has a Pulsar consumer trigger, which listens to the same topic using a Pulsar subscription and receives the batch of messages together.
```

Comment:
You can just describe more about batching and chunking here and for application and flow you can add detailed description in the Flow section.